planner/core/explain.go

```diff
@@ -246,6 +248,15 @@ func (p *PhysicalTableScan) OperatorInfo(normalized bool) string {
 	if p.StoreType == kv.TiFlash && p.Table.GetPartitionInfo() != nil && p.IsMPPOrBatchCop && p.ctx.GetSessionVars().StmtCtx.UseDynamicPartitionPrune() {
 		buffer.WriteString(", PartitionTableScan:true")
 	}
+	if len(p.runtimeFilterList) > 0 {
+		buffer.WriteString(", runtime filter:")
+		for i, runtimeFilter := range p.runtimeFilterList {
+			if i != 0 {
+				buffer.WriteString(", ")
+			}
+			buffer.WriteString(runtimeFilter.ExplainInfo(false))
+		}
+	}
```

Comment:
Just to remind, any changes in the content will cause a different plan digest and therefore be considered as a different execution plan by modules like stmt summary.

Reply:
Are there any other effects of different execution plan? With rf and without rf itself is a different planning plan